ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py

```diff
@@ -75,20 +95,39 @@ class OpenMetadataLineageBackend(LineageBackend):
     auth_provider_type = no-auth # use google here if you are
         configuring google as SSO
     secret_key = google-client-secret-key # it needs to be configured
-        only if you are using google as SSO
+        only if you are using google as SSO the one configured in openMetadata
+    openmetadata_api_endpoint = http://localhost:8585
+    auth_provider_type = no-auth # use google here if you are configuring google as SSO
+    secret_key = google-client-secret-key # it needs to be configured
+                 only if you are using google as SSO
     """
 
     def __init__(self) -> None:
+        """
+        Instantiate a superclass object and run lineage config function
+        """
         super().__init__()
         _ = get_lineage_config()
 
+    # pylint: disable=protected-access
     @staticmethod
     def send_lineage(
         operator: "BaseOperator",
         inlets: Optional[List] = None,
         outlets: Optional[List] = None,
         context: Dict = None,
     ) -> None:
+        """
+        Send lineage to OpenMetadata
+
+        Args
+            operator (BaseOperator):
```

Comment:
I'm curious if there is much value in having docstring args without any description

Reply:
That is a good question. My rational was based on 2 elements:
1. It still shows up in IDEs
2. It acts as a place holder so that people with more info can add to the docstring (as I missed som context on some instance)